docs/README.md

```diff
@@ -23,7 +23,7 @@ Use the Hugo shortcode [relref](https://gohugo.io/content-management/cross-refer
 
 ### Edit the side menu
 
-Edit [sources/menu.yaml](sources/menu.yaml) to make changes to the sidebar. Restart the `make run` command for changes to take effect.
+Edit [sources/menu.yaml](sources/menu.yaml) to make changes to the sidebar. Restart the `make docs` command for changes to take effect.
```

Comment:
What do you mean by "restart"? How do I do that? 

Remember that this is likely to be read by writers, so we need to make the instructions explicit and simple.

Also, do we run this in PowerShell or the cmd window?

Reply:
> Also, do we run this in PowerShell or the cmd window?

The command should be run in the `docs` directory.

Reply:
On my Windows machine, will I need to run this in Powershell or in the CMD window?

Also, what do you mean by "restart"? How do I do that?

Remember that this is likely to be read by writers, so we need to make the instructions explicit and simple. If you make the command something they can copy/paste, that would be ideal.

Reply:
> On my Windows machine, will I need to run this in Powershell or in the CMD window?

Wherever you run your commands.

> Also, what do you mean by "restart"? How do I do that?

Stop the process and run it again.